lib/mondo/merchant.rb

```diff
@@ -1,7 +1,7 @@
 module Mondo
   class Merchant < Resource
 
-    attr_accessor :id, :group_id, :logo, :name, :raw_data, :address, :emoji
+    attr_accessor :id, :group_id, :logo, :name, :raw_data, :address, :emoji, :online
```

Comment:
I wonder if you could define `bool_accessor` that creates a method called `online?`.

Same with `is_load?` and `settled?` below